Solitaire.java

```diff
@@ -167,12 +167,23 @@ public static boolean playAgain(Scanner s){
 	* @param e the formatted array from format()
 	*/
 
-	public static void encrypt(String[] e) {
-		//convert letters to numbers
-		//get keystream values
-		//add the numbers together
-		//if the added numbers are greater than 26, subtract by 26
-		//convert back to letters
+	public static void encrypt(String[] e, int [] key) {
+		int[] intArray = new int[e.length()];
+		for(int i = 0; i < e.length(); i++){
+			char letter = e[i].charAt(0);
+			int ascii = (int) letter;
+			ascii = ascii - 'A' + 1;
+			intArray[i] = ascii;
+		}
+		int[] finalConvert = new int[intArray.length()];
+		char[] finalLetters = new char[intArray.length()];
+		for(int i=0; i < finalConvert.length(); i++){
+			finalConvert[i] = intArray[i] + key[i];
+			if(finalConvert[i] > 26){ //if greater than 26, subtract 26
+				finalConvert[i] = finalConvert[i] - 26;
+			}
+			finalLetters[i] = finalConvert[i] + 'A' - 1; //convert the numbers back to letters
```

Comment:
I ran this through the compiler and it's saying that there's a possible lossy conversion from int to char. Here you might have to use a loop similar to what you used to change the chars into ints.